### PR TITLE
updated Japanese localizations

### PR DESCRIPTION
**Pull request by ssogabe**

### CODE REVIEW

**Comment by oleg-nenashev** [comment]
CC @kohsuke and @ikedam , maybe you could review it

File: src/main/resources/hudson/maven/MavenModuleSet/configure-entries_ja.properties [hunk 1]
```diff
@@ -51,3 +49,16 @@ runIfStable=\u30d3\u30eb\u30c9\u304c\u6210\u529f\u3057\u305f\u5834\u5408\u306e\u
...
+Settings\ file=\u8a2d\u5b9a\u30d5\u30a1\u30a4\u30eb
+Global\ Settings\ file=\u30b0\u30ed\u30fc\u30d0\u30eb\u8a2d\u5b9a\u30d5\u30a1\u30a4\u30eb
+Disable\ automatic\ fingerprinting\ of\ consumed\ and\ produced\ artifacts=\
+  \u4f7f\u7528\u3057\u305f\u3082\u3057\u304f\u306f\u751f\u6210\u3057\u305f\u6210\u679c\u7269\u306e\u6307\u7d0b\u3092\u81ea\u52d5\u53d6\u5f97\u3059\u308b\u3053\u3068\u3092\u7121\u52b9\u5316\u3059\u308b
+Disable\ automatic\ site\ documentation\ artifact\ archiving=\
+  \u30b5\u30a4\u30c8\u6587\u66f8\u306e\u81ea\u52d5\u751f\u6210\u3092\u7121\u52b9\u5316\u3059\u308b
+Block\ downstream\ trigger\ when\ building=\
+  \u4e0b\u6d41\u30d7\u30ed\u30b8\u30a7\u30af\u30c8\u306e\u30d3\u30eb\u30c9\u3092\u30d6\u30ed\u30c3\u30af\u3059\u308b
+Enable\ triggering\ of\ downstream\ projects=\
+   \u4e0b\u6d41\u30d7\u30ed\u30b8\u30a7\u30af\u30c8\u306e\u30d3\u30eb\u30c9\u3092\u6709\u52b9\u5316\u3059\u308b
```
Comment (ikedam) [inline src/main/resources/hudson/maven/MavenModuleSet/configure-entries_ja.properties]: Decoded tests:
```
Schedule\ build\ when\ some\ upstream\ has\ no\ successful\ builds=\
  上流で成功していないジョブがある場合でもスケジュールする
Maven\ Version.error.3=<a href="{0}/configureTools" target="_new">グローバルツールの設定</a>で設定してください。
Settings\ file=設定ファイル
Global\ Settings\ file=グローバル設定ファイル
Disable\ automatic\ fingerprinting\ of\ consumed\ and\ produced\ artifacts=\
  使用したもしくは生成した成果物の指紋を自動取得することを無効化する
Disable\ automatic\ site\ documentation\ artifact\ archiving=\
  サイト文書の自動生成を無効化する
Block\ downstream\ trigger\ when\ building=\
  下流プロジェクトのビルドをブロックする
Enable\ triggering\ of\ downstream\ projects=\
   下流プロジェクトのビルドを有効化する
```

File: src/main/resources/hudson/maven/Messages_ja.properties [hunk 2]
```diff
@@ -20,48 +20,50 @@
...
-RedeployPublisher.RepositoryURL.Mandatory=\u30EA\u30DD\u30B8\u30C8\u30EAURL\u306F\u5FC5\u9808\u3067\u3059\u3002
-ReleaseAction.DisplayName=\u65B0\u898F\u30D0\u30FC\u30B8\u30E7\u30F3\u306E\u30EA\u30EA\u30FC\u30B9
+RedeployPublisher.getDisplayName=Maven\u30ea\u30dd\u30b8\u30c8\u30ea\u3078\u6210\u679c\u7269\u3092\u30c7\u30d7\u30ed\u30a4
+ReleaseAction.DisplayName=\u65b0\u898f\u30d0\u30fc\u30b8\u30e7\u30f3\u306e\u30ea\u30ea\u30fc\u30b9
+MavenModuleSet.readResolve_updateConfigProvider=\u30a8\u30e9\u30fc: 'Config File Provider Plugin'\u3092\u30a2\u30c3\u30d7\u30c7\u30fc\u30c8\u3057\u3066\u304f\u3060\u3055\u3044\u3002\u73fe\u5728\u306e\u30d0\u30fc\u30b8\u30e7\u30f3\u306f\u3053\u308c\u4ee5\u4e0a\u30b5\u30dd\u30fc\u30c8\u3055\u308c\u307e\u305b\u3093(id={0})\u3002
+MavenModuleSet.readResolve_missingConfigProvider=\u30a8\u30e9\u30fc: 'Config File Provider Plugin'\u304c\u30a4\u30f3\u30b9\u30c8\u30fc\u30eb\u3055\u308c\u3066\u3044\u306a\u3044\u304b\u3001\u7121\u52b9\u5316\u3055\u308c\u3066\u3044\u307e\u3059\u3002\
+\u305d\u306e\u305f\u3081\u3001\u8a2d\u5b9a\u3092\u5b8c\u5168\u306b\u8aad\u307f\u8fbc\u3080\u3053\u3068\u304c\u3067\u304d\u307e\u305b\u3093\u3002
```
Comment (ikedam) [inline src/main/resources/hudson/maven/Messages_ja.properties]: Decoded texts:
```
RedeployPublisher.getDisplayName=Mavenリポジトリへ成果物をデプロイ
ReleaseAction.DisplayName=新規バージョンのリリース
MavenModuleSet.readResolve_updateConfigProvider=エラー: 'Config File Provider Plugin'をアップデートしてください。現在のバージョンはこれ以上サポートされません(id={0})。
MavenModuleSet.readResolve_missingConfigProvider=エラー: 'Config File Provider Plugin'がインストールされていないか、無効化されています。\
そのため、設定を完全に読み込むことができません。
```

File: src/main/resources/hudson/maven/reporters/MavenMailer/config_ja.properties [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
+Recipients=\u53d7\u4fe1\u8005
+Send\ e-mail\ for\ every\ unstable\ build=\u4e0d\u5b89\u5b9a\u30d3\u30eb\u30c9\u306b\u30e1\u30fc\u30eb\u3092\u9001\u4fe1\u3059\u308b
+Send\ e-mail\ for\ each\ failed\ module=\u5931\u6557\u3057\u305f\u30e2\u30b8\u30e5\u30fc\u30eb\u6bce\u306b\u30e1\u30fc\u30eb\u3092\u9001\u4fe1\u3059\u308b
+Send\ separate\ e-mails\ to\ individuals\ who\ broke\ the\ build=\
+  \u30d3\u30eb\u30c9\u3092\u58ca\u3057\u305f\u958b\u767a\u8005\u500b\u4eba\u3054\u3068\u306b\u30e1\u30fc\u30eb\u3092\u9001\u4fe1\u3059\u308b
```
Comment (ikedam) [inline src/main/resources/hudson/maven/reporters/MavenMailer/config_ja.properties]: Decoded texts:
```
Recipients=受信者
Send\ e-mail\ for\ every\ unstable\ build=不安定ビルドにメールを送信する
Send\ e-mail\ for\ each\ failed\ module=失敗したモジュール毎にメールを送信する
Send\ separate\ e-mails\ to\ individuals\ who\ broke\ the\ build=\
  ビルドを壊した開発者個人ごとにメールを送信する
```

File: src/main/resources/hudson/maven/MavenModuleSet/configure-entries_ja.properties (hunk 1 above)
Comment (ikedam) [inline src/main/resources/hudson/maven/MavenModuleSet/configure-entries_ja.properties]: 「の自動生成を無効化する」 のほうが「を自動取得することを無効化する」よりも短くわかりやすいので、そちらに合わせていただくのが良いと思います。

File: src/main/resources/hudson/maven/Messages_ja.properties [hunk 4]
```diff
@@ -20,48 +20,50 @@
...
-MavenBuilder.AsyncFailed=\u975E\u540C\u671F\u5B9F\u884C\u5931\u6557
-MavenBuilder.Failed=Maven\u306F\u30A8\u30E9\u30FC\u3067\u5931\u6557\u3057\u307E\u3057\u305F\u3002
-MavenBuilder.Waiting=Jenkins\u304C\u30C7\u30FC\u30BF\u53CE\u96C6\u3092\u5B8C\u4E86\u3059\u308B\u307E\u3067\u5F85\u6A5F\u4E2D
+MavenBuild.FailedEarlier=\u3053\u306e\u30e2\u30b8\u30e5\u30fc\u30eb\u306e\u524d\u306b\u30d3\u30eb\u30c9\u304c\u5931\u6557\u3057\u307e\u3057\u305f
+MavenBuild.KeptBecauseOfParent={0}\u304c\u4fdd\u7559\u4e2d\u306e\u305f\u3081\u4fdd\u7559\u3057\u307e\u3059
+MavenBuild.Triggering={0}\u306e\u65b0\u898f\u30d3\u30eb\u30c9\u306e\u5b9f\u884c
+MavenBuilder.Aborted=\u4e2d\u6b62
+MavenBuilder.AsyncFailed=\u975e\u540c\u671f\u5b9f\u884c\u5931\u6557
+MavenBuilder.Failed=Maven\u306f\u30a8\u30e9\u30fc\u3067\u5931\u6557\u3057\u307e\u3057\u305f\u3002
+MavenBuilder.Waiting=Jenkins\u304c\u30c7\u30fc\u30bf\u53ce\u96c6\u3092\u5b8c\u4e86\u3059\u308b\u307e\u3067\u5f85\u6a5f\u4e2d
```
Comment (ikedam) [inline src/main/resources/hudson/maven/Messages_ja.properties]: -1 for unnecessary changes. It makes difficult to review and track changes.

File: src/main/webapp/blockTriggerWhenBuilding_ja.html [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+        By default when a newer build of an upstream project is building (or in queue),
+        an earlier build will not trigger builds of downstream modules.
+        This is under the assumption that those builds would soon be superseded and are not of interest.
+    </p>
+    <p>
+        You can however disable this logic on the upstream project, which will result in a fuller sequence of downstream builds.
+        This is not be confused with <b>Block build when upstream project is building</b> (marked on the other project),
+        which controls <em>scheduling</em> of the downstream project once it has been triggered.
+    </p>
+</div>
```
Comment (ikedam) [inline src/main/webapp/blockTriggerWhenBuilding_ja.html]: Please don't add untranslated files. I believe it prevents automatic detection for untranslated texts.

File: src/main/webapp/downstreamDisabled_ja.html [hunk 6]
```diff
@@ -1,3 +1,3 @@
 <div>
-  このオプションをチェックすると、下流プロジェクトのジョブをスケジュールしません。
+  このオプションをチェックすると、下流プロジェクトのジョブをスケジュールします。
```
Comment (ikedam) [inline src/main/webapp/downstreamDisabled_ja.html]: Note: 924c274a497d76ac4958ddebad30759f2cd6a5b6
Other translations should have the same problem...

**Comment by olamy** [comment]
old PR without needed fixes so closing it.
If you still need feel free to reopen and update it
